src/App.tsx

```diff
@@ -1,11 +1,30 @@
+import { RouterProvider, createBrowserRouter } from 'react-router-dom';
 import { ThemeProvider } from 'styled-components';
 
+import ApplicationPage from './pages/ApplicationPage';
+import LoginPage from './pages/LoginPage';
+import MainPage from './pages/MainPage';
+import ModelInfoPage from './pages/ModelInfoPage';
+import MyPage from './pages/MyPage';
+import OfferInfoPage from './pages/OfferInfoPage';
+import SignUpPage from './pages/SignUpPage';
 import GlobalStyle from './styles/GlobalStyle';
 import theme from './styles/Theme';
 
+const router = createBrowserRouter([
+  { path: '/', element: <MainPage /> },
+  { path: '/login', element: <LoginPage /> },
+  { path: '/signup', element: <SignUpPage /> },
+  { path: '/modelinfo', element: <ModelInfoPage /> },
+  { path: '/application', element: <ApplicationPage /> },
+  { path: '/offerinfo', element: <OfferInfoPage /> },
```

Comment:
일반적으로 path에서 단어 분리를 할 땐 케팝케이스(공백을 하이픈으로 구분하는 방식)로 합니다! 
```suggestion
  { path: '/sign-up', element: <SignUpPage /> },
  { path: '/model-info', element: <ModelInfoPage /> },
  { path: '/application', element: <ApplicationPage /> },
  { path: '/offer-info', element: <OfferInfoPage /> },
```
요렇게 바꿔줘도 좋을 것 같네요!_!